aten/src/ATen/cpu/vec256/vec256_int.h

```diff
@@ -0,0 +1,121 @@
+#pragma once
```

Comment:
You could also merge float and double into _floating just as with _int

Or we could also split on avx and avx2

Reply:
It's split based on data type: e.g. `_m256i` vs. `_m256` vs. `_m256d`.

As a matter of practicality, it makes it easy to compare float vs. double and I expect these specializations to have the most functions added to it.